README.md

```diff
@@ -229,13 +319,23 @@ UIKit lets you align labels by top, bottom, or baseline. BonMot includes `BONTex
 
 You can use `BONTextAlignmentConstraint` programmatically or in Interface Builder. In code, use the convenience initializer:
 
+```swift
+BONTextAlignmentConstraint(item: someLabel,
+                      attribute: BONConstraintAttribute.CapHeight,
```

Comment:
@ZevEisenberg I was considering that, but decided to include the enum name since I thought the user might be led at first glance to believe (based on the function's similarity to `NSLayoutConstraint`'s init) that they could use a `NSLayoutConstraint` here.